plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/xtext/server/XLanguageServerImpl.java

```diff
@@ -445,8 +445,7 @@ public void exit() {
 				LOG.info("Shutdown done");
 				return result;
 			};
-		});
-		return CompletableFuture.completedFuture(null);
+		}).thenApply((any) -> new Object());
```

Comment:
I guess the `#thenApply()` is only needed to satisfy the return type of `#shutdown()`?